GitUI/UserEnvironmentInformation.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Windows.Forms;
+using GitCommands;
+
+namespace GitUI
+{
+    public sealed class UserEnvironmentInformation
+    {
+        public static void CopyInformation()
+        {
+            Clipboard.SetText(GetInformation());
+        }
+
+        public static string GetInformation()
+        {
+            StringBuilder sb = new StringBuilder();
```

Comment:
I added the newlines surrounding the block to prevent the problem of the user pasting and the markup not being correct (blank line before and after list).  Depending on what they highlight and paste the blank line can be removed in the issue. I am not bothered by the removal.  Just wanted to make you aware of the reason why I had it in there.